### DISCUSSION TITLE
[Script request] SearXNG

### DISCUSSION THREAD

**Discussion by DSDV** [community-scripts/ProxmoxVE#122] [Request script]
### Application Name

SearXNG

### Website

https://github.com/searxng/searxng

### Description

there where a lot of requests, 

so i decided to open discussion here even if it may just become a place to sort things out. 
Apparently the official documentation is not good. so it is hard to build a scrip lets try to be patient and understanding <3 

Here is an attempt from a community member: 
https://github.com/tteck/Proxmox/discussions/318#discussioncomment-11128753

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by yry-dev** [comment]
Has anyone looked at this one recently?  I might make an attempt at it.

**Reply by MickLesk** [reply]
You're welcome to take a look, but the script will probably be damn hard. I've looked at it a few times before.

**Reply by jproz** [reply]
> You're welcome to take a look, but the script will probably be damn hard. I've looked at it a few times before.

https://github.com/tteck/Proxmox/discussions/318#discussioncomment-11128753

Just to highlight your prior comment and the response from the prior tteck archive. 

I agree hard. Wonder if the work that the responding party could be adapted.  
<shrug>

**Comment by Solear** [comment]
Up, because whooge faces some google issues :-(

**Reply by knario** [reply]
Hi, I installed searchXNG directly on casaOs and it goes very well and smoothly.
I know it's not the most suitable way, but it's a good idea until Whoogle releases some update and the problem is fixed.

**Reply by Solear** [reply]
> Hi, I installed searchXNG directly on casaOs and it goes very well and smoothly. I know it's not the most suitable way, but it's a good idea until Whoogle releases some update and the problem is fixed.

Thank you. In the meantime Ive installed a dockge lxc and spin up a searxng compose up.

**Comment by kiwinh0** [comment]
+1

**Comment by Matiusauce** [comment]
Yes please! Would be cool if we had the option to add in one of the perplexity clones that piggyback off searxng like perplexica for example.

**Comment by michelroegl-brunner** [comment]
Merged.

**Comment by SoTHISIsFitness** [comment]
This can probably be moved from 'Deferred' to 'Completed' on the project kanban since... well... a SearXNG helper script was created and published to the site 😅 works great! lol

**Reply by TegridyPharm** [reply]
I must be doing something wrong - I've tried using the community script and it crashes saying it can't find an lxml package for python.  I've verified that lxml is there (pip show lxml), but it's just not working out.

Anyone know what this might be due to or if something recent changed?

**Reply by MickLesk** [reply]
Last week it works, create an correct issue an someone can take a look

**Reply by TegridyPharm** [reply]
> Last week it works, create an correct issue an someone can take a look

Done - created issue #7424.  Thank you.

**Comment by Tamalero** [comment]
Hi folks! sorry to barge for this. Is there a proper way to update this container? or should I just reinstall?
